### PR TITLE
Added support to send events to a pubsub topic

### PR DESCRIPTION
**Pull request by isaldana**
This addresses #222.

### CODE REVIEW

**Comment by isaldana** [comment]
It looks like the lint test is failing because the the `Dockerfile` needs to be updated to freeze ` golangci/golangci-lint:v1.32 AS linter` since the newest version breaks the existing code (even without my PR). I updated it in this PR.

**Comment by fsouza** [comment]
Hi  @isaldana thanks for contributing and apologies for such a delay to get back to you. I'll look into this PR later today or tomorrow!

File: internal/config/config.go [hunk 1]
```diff
@@ -17,27 +17,35 @@ import (
...
+	allowedCORSHeaders  []string
+	scheme              string
+	host                string
+	port                uint
+	backend             string
+	fsRoot              string
+	eventPubsubProjecID string
+	eventPubsubTopic    string
+	eventPrefix         string
+	eventList           []string
```
Comment (fsouza) [inline internal/config/config.go:40]: Can you move these fields into a dedicated struct for events?

Comment (fsouza) [reply internal/config/config.go:40]: (also, with the growing number of flags we have, we'd should probably consider adding some form of config file, not a blocker for this PR though, definitely something I'll have to figure out later)

**Review by fsouza — COMMENTED** [review]
Sorry for such a terrible turnaround on this. I've rebased it and gave it a first pass. Looks good overall, still thinking about the idea though. It'd be good if this could be 100% mocked or something like that (perhaps we can leverage the eventPublisher interface for more than just tests). Since this is all internal state, we don't need to block the PR on that decision.

Also, if you've moved on to work on other stuff, let me know and I pickup the changes to get this merged!

**Comment by fsouza** [comment]
Closing in favor of #490.

**Pull request closed without merging**